assessment/asr_validation/wald2017B/analysis/asr_concrete_block_biaxial_rebar.i

```diff
@@ -0,0 +1,1070 @@
+    rank_two_tensor = asr_expansion
+    variable = ASR_strain_yz
+    index_i = 1
+    index_j = 2
+    execute_on = 'timestep_end'
+  [../]
+
+  [./ASR_strain_zx]
+    type = RankTwoAux
+    block = 1
+    rank_two_tensor = asr_expansion
+    variable = ASR_strain_zx
+    index_i = 0
+    index_j = 2
+    execute_on = 'timestep_end'
+  [../]
+  [./thermal_strain_xx]
+    type = RankTwoAux
+    block = 1
+    rank_two_tensor = thermal_expansion
+    variable = thermal_strain_xx
+    index_i = 0
+    index_j = 0
+    execute_on = 'timestep_end'
+  [../]
+  [./thermal_strain_yy]
+    type = RankTwoAux
+    block = 1
+    rank_two_tensor = thermal_expansion
+    variable = thermal_strain_yy
+    index_i = 1
+    index_j = 1
+    execute_on = 'timestep_end'
+  [../]
+  [./thermal_strain_zz]
+    type = RankTwoAux
+    block = 1
+    rank_two_tensor = thermal_expansion
+    variable = thermal_strain_zz
+    index_i = 2
+    index_j = 2
+    execute_on = 'timestep_end'
+  [../]
+
+  [./volumetric_strain]
+    type = RankTwoScalarAux
+    scalar_type = VolumetricStrain
+    rank_two_tensor = total_strain
+    variable = volumetric_strain
+    block = 1
+  [../]
+
+  [./k]
+    type = MaterialRealAux
+    variable = thermal_conductivity
+    property = thermal_conductivity
+    execute_on = 'timestep_end'
+    block = 1
+  [../]
+  [./capacity]
+    type = MaterialRealAux
+    variable = thermal_capacity
+    property = thermal_capacity
+    execute_on = 'timestep_end'
+    block = 1
+  [../]
+
+  [./rh_capacity]
+    type = MaterialRealAux
+    variable = moisture_capacity
+    property = moisture_capacity
+    execute_on = 'timestep_end'
+    block = 1
+  [../]
+  [./rh_duff]
+    type = MaterialRealAux
+    variable = humidity_diffusivity
+    property = humidity_diffusivity
+    execute_on = 'timestep_end'
+    block = 1
+  [../]
+  [./wc_duff]
+    type = MaterialRealAux
+    variable = water_content
+    property = moisture_content
+    execute_on = 'timestep_end'
+    block = 1
+  [../]
+  [./hydrw_duff]
+    type = MaterialRealAux
+    variable = water_hydrated
+    property = hydrated_water
+    execute_on = 'timestep_end'
+    block = 1
+  [../]
+
+  [damage_index]
+    type = MaterialRealAux
+    block = 1
+    variable = damage_index
+    property = damage_index
+    execute_on = timestep_end
+  []
+  [./areax]
+    type = ConstantAux
+    block = '2'
+    variable = area
+    value = 1.33e-4
+    execute_on = 'initial timestep_begin'
+  [../]
+  [./areaz]
+    type = ConstantAux
+    block = '3'
+    variable = area
+    value = 1.33e-4
+    execute_on = 'initial timestep_begin'
+  [../]
+  [./axial_stress]
+    type = MaterialRealAux
+    block = '2 3'
+    variable = axial_stress
+    property = axial_stress
+  [../]
+[]
+
+[Functions]
+  [./ramp_temp]
+    type = PiecewiseLinear
+    x = ' 0.0 2722893.36 3498624.13 4274354.9 5050085.66 5825816.43 6566286.71 7306756.99 8082487.76 8858218.53 9633949.3 10409680.1 11185410.8 11961141.6
```

Comment:
I think these temperature and humidity functions are shared across multiple cases, aren't they?  If so, I would prefer that we have them defined in csv files that we read in for these functions.